src/components/action-group/action-group.ts

```diff
@@ -1,9 +1,9 @@
 import { CSSResultGroup, html, LitElement, TemplateResult, PropertyValues } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
-import { SbbButtonSize } from '../button';
+import { SbbButtonElement, SbbButtonSize } from '../button';
 import { SbbHorizontalFrom, SbbOrientation } from '../core/interfaces';
-import { SbbLinkSize } from '../link';
+import { SbbLinkElement, SbbLinkSize } from '../link';
```

Comment:
it's important to only import types, otherwise the whole component will be loaded
```suggestion
import type { SbbLinkElement, SbbLinkSize } from '../link';
```